### DISCUSSION TITLE
[Feature Request] Stock Verification/Validation

### DISCUSSION THREAD

**Discussion by ramalmaty** [Blair2004/NexoPOS#808] [Ideas]
Hi,
Overall the program is very good!
Only the function lacking is the Stock verification what the shop owners usually do.
Usually what happens is there will be a page which collects the barcode scans, and finally give how many times the barcode is scanned and tally with the current available stock and give the differed stock data which didn't mach with the Inventory(Current stock).
This is a must for the stock owners to verify at regular intervals.

Usually the page contains:
LogID,User, Date & Time,Warehouse/Shop Number, Status. Can also have total number of items Available in store/scanned.
Detailed list should have the product wise available & Scanned.

At the time of stock verification POS should not be accessible for that shop.

Hope, you will consider this feature.

Thank you

Regards
Ram

**Comment by mbugikp** [comment]
I think what @ramalmaty is saying is that we need a stock status report, the stock status report shows all the current items with the current quantities per item e.g. Black trouser 5pcs, Blue shirt 16pcs etc, the report normally adds the cost price and selling price to the items, then calculates the current stock according to cost and selling price, it is very useful when doing stock count or stock take.

**Reply by ramalmaty** [reply]
Example: 
One of the scenarios:
Every month before giving salary, shop owners will calculate the real available stock to tally with the inventory stock in program.

Usually in other programs  how it is done:
There will be a module to conduct inventory.
1) It will set sales/POS sales to pause and keep the program in inventory mode.
2) There will be the list of all products in stock with columns Stock & actual.
3) the operator/Owner/Responsible person keeps scanning the goods barcodes, the program keep filling the actual, depending on how many times the code is scanned, it keeps adding the actual.
4) in the end, the difference between the stock and actual is checked and necessary action is taken.
Reasons of difference can be, unapproved stock transfer by shopkeeper, stolen goods, shopkeeper physically got goods but not approved in program etc...
5) Then this will go in to adjustments tab and fixed there.
6) There should be an option to edit the Quantity if some product is wrongly/accidentally scanned double times.

**Reply by Blair2004** [reply]
Approved
